<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font: Open Sans SemiBold;
	font-color: white;
}


QPushButton{
    background-color: #2e9cca;
    border-style: solid;
    border-width: 2px;
    border-radius: 10px;
    border-color: #aaabb8;
	color: white;
}

QPushButton:pressed {
	border-width: 5px;
	border-color: white;
	
}

QPushButton:hover{
	background-color: #2964BA;
}

QPushButton:disabled{
	background-color: #464866
}

#centralwidget{
	background: #2A1B3D;
}

QLineEdit{
	background-color: #aaabb8;
	border-style: solid;
	border-width: 2px;
	border-radius: 10px;
	border-color: white;
	selection-background-color: dark grey;
	color: black;
	padding: 2px
}

QLabel{
	color: white;
	font: bold;

}

QTableWidget{
	background-color: #464866;
	color: white;
	selection-background-color: #2964BA;
	border-color: #aaabb8;
	border-style: solid;
	border-width: 1px;
	border-radius: 15px;
	gridline-color: #aaabb8;
}

QRadioButton{
	color: white;
}

QComboBox{
	background: #464866;
	color: white;
	border-color: #aaabb8;
	border-style: solid;
	border-width: 3px;
	border-radius: 10px;
	font-size: 16px;
}

QComboBox:hover{
	background: #2964BA
}



</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans SemiBold</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Create a Question</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans SemiBold</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Question Type</string>
    </property>
   </widget>
   <widget class="QComboBox" name="q_operator_dropdown">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans SemiBold</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Addition</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Subtraction</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Inverse</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Multiply</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Determinant</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="q_check_valid">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>480</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Check Valid</string>
    </property>
   </widget>
   <widget class="QPushButton" name="q_add_question">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>480</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Question</string>
    </property>
   </widget>
   <widget class="QPushButton" name="q_finish_quiz">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>480</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Finish Quiz</string>
    </property>
   </widget>
   <widget class="QComboBox" name="q_dimension_dropdown">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>120</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans SemiBold</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <item>
     <property name="text">
      <string>3x3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2x2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4x4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5x5</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="q_auto_generate">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>120</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto-generate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="q_update">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>120</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QWidget" name="q_left_matrix" native="true">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>271</width>
      <height>201</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="q5">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="q2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="q7">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="q3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="l2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>0</y>
       <width>21</width>
       <height>201</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="l3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="l4">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>-10</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="q8">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="l6">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>190</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="l5">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>0</y>
       <width>20</width>
       <height>201</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="q1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="l1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>-10</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="q6">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="q4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="q9">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="q_right_matrix" native="true">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>190</y>
      <width>411</width>
      <height>231</height>
     </rect>
    </property>
    <widget class="Line" name="line_18">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>20</width>
       <height>201</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="q15">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="q17">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>170</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="q14">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="q18">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>170</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="q16">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line_17">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>210</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_13">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="q12">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="q13">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line_14">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="q11">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="q10">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="q_operator_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Sans SemiBold</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font-size: 40px;
}</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="Line" name="line_15">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>210</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_16">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>21</width>
       <height>201</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="q_cancel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
